{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"name":"Untitled0.ipynb","provenance":[],"mount_file_id":"1TQdedKz-rjwQgskYbCiA5uaV1oxVXObR","authorship_tag":"ABX9TyMbR2h2OqmQ5/vExTqCHvZq"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"}},"cells":[{"cell_type":"code","metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"iqRNjSETyHu8","executionInfo":{"status":"ok","timestamp":1618494574326,"user_tz":-330,"elapsed":5088,"user":{"displayName":"aayush jain","photoUrl":"","userId":"06466801638325838312"}},"outputId":"e88ac351-ff8b-4193-cc48-06284198e00e"},"source":["!python3 \"/content/drive/MyDrive/GenieTalk/Ticket_classification/final_deliver/Flaskapp/app.py\""],"execution_count":1,"outputs":[{"output_type":"stream","text":["2021-04-15 13:49:34.792957: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0\n","Traceback (most recent call last):\n","  File \"/content/drive/MyDrive/GenieTalk/Ticket_classification/final_deliver/Flaskapp/app.py\", line 11, in <module>\n","    model = tf.keras.models.load_model(\"fastTextmodel.h5\")\n","  File \"/usr/local/lib/python3.7/dist-packages/tensorflow/python/keras/saving/save.py\", line 211, in load_model\n","    loader_impl.parse_saved_model(filepath)\n","  File \"/usr/local/lib/python3.7/dist-packages/tensorflow/python/saved_model/loader_impl.py\", line 114, in parse_saved_model\n","    constants.SAVED_MODEL_FILENAME_PB))\n","OSError: SavedModel file does not exist at: fastTextmodel.h5/{saved_model.pbtxt|saved_model.pb}\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"QisKFIK8yzrE"},"source":[""],"execution_count":null,"outputs":[]}]}